Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1605,7 +1612,7 @@
     <!-- mijenja u korist češće riječi -->
     <RegEx find="([vV])rača" replaceWith="$1raća" />
     <RegEx find="zleči" replaceWith="zliječi" />
-    <RegEx find="(?&lt;!obra)zovati" replaceWith="zirati" />
+    <RegEx find="(?&lt;!([Oo])bra)zova([lt])" replaceWith="zira$2" />
```

Comment:
Maybe also "n" and "o", `zova([lnot])`, for e.g. "organizovan" and "organizovao".